hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -447,6 +447,12 @@
       .sinceVersion("0.11.0")
       .withDocumentation("Master control to disable all table services including archive, clean, compact, cluster, etc.");
 
+  public static final ConfigProperty<Boolean> RELEASE_RESOURCE_ENABLE = ConfigProperty
+      .key("hoodie.release.resource.enabled")
```

Comment:
can we name this "hoodie.release.resource.on.completion.enable"

Reply:
yes, the "hoodie.release.resource.on.completion.enable"  more clear